migrate_test.go

```diff
@@ -432,7 +525,7 @@ func (s *SqliteMigrateSuite) TestPlanMigrationWithHoles(c *C) {
 	c.Assert(plannedMigrations[2].Queries[0], Equals, up)
 
 	// first catch up to current target state 123, then migrate down 1 step to 12
-	plannedMigrations, _, err = PlanMigration(s.Db, "sqlite3", migrations, Down, 1)
+	plannedMigrations, _, err = PlanMigration(s.Db, "sqlite3", migrations, Down, 1, -1)
```

Comment:
Needing to change existing tests is a good indicator that you'll be breaking other people's code (which they won't like).

Ideally only new tests are added, no other changes to tests required.